src/Goerli-DssSpell.sol

```diff
@@ -41,29 +45,98 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     //    https://ipfs.io/ipfs/QmPgPVrVxDCGyNR5rGp9JC5AUxppLzUAqvncRJDcxQnX1u
     //
 
-    uint256 constant TWO_TWO_FIVE_PCT_RATE = 1000000000705562181084137268;
-    uint256 constant FOUR_PCT_RATE         = 1000000001243680656318820312;
-
     // Turn office hours off
     function officeHours() public override returns (bool) {
         return false;
     }
 
+    address constant MCD_END   = 0xb82F60bAf6980b9fE035A82cF6Acb770C06d3896;
+    address constant MCD_CURE  = 0xFA5d993DdA243A57eefbbF86Cb3a1c817Dfc7e4E;
+    address constant MCD_FLASH = 0xAa5F7d5b29Fa366BB04F6E4c39ACF569d5214075;
+
+    address immutable MCD_VAT          = DssExecLib.vat();
+    address immutable MCD_VOW          = DssExecLib.vow();
+    address immutable MCD_CAT          = DssExecLib.cat();
+    address immutable MCD_DOG          = DssExecLib.dog();
+    address immutable MCD_SPOT         = DssExecLib.spotter();
+    address immutable MCD_POT          = DssExecLib.pot();
+    address immutable MCD_ESM          = DssExecLib.esm();
+    address immutable ILK_REGISTRY     = DssExecLib.reg();
+    address immutable MCD_END_OLD      = DssExecLib.end();
+    address immutable MCD_FLASH_LEGACY = DssExecLib.getChangelogAddress("MCD_FLASH");
+
     function actions() public override {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralOnboardingAction
-        onboardNewCollaterals();
+        // onboardNewCollaterals();
+
+        // -------------------------- Cure + New End ---------------------------
+        DssExecLib.setValue(MCD_END, "wait", EndAbstract(MCD_END_OLD).wait());
```

Comment:
Consider adding a sanity check that ES is not underway (check `End.live() == 1`). Otherwise I think we can revert.
This is to prevent interfering with shutdown (deauthing old End).

Reply:
We have never done it in prev `End` replacements, but it makes total sense to add it.
Done now.